crates/proof-of-sql/src/proof_primitive/dory/pack_scalars.rs

```diff
@@ -165,6 +165,35 @@ fn pack_bit<const LEN: usize, T: OffsetToBytes<LEN>>(
     });
 }
 
+fn pack_fixed_size_binary(
```

Comment:
This code is mostly just duplicating `pack_bit`. Instead, I believe it should be doable to make the `column` parameter in `pack_bit` be an `impl IntoIterator<T>` and `impl OffsetToBytes for &[u8]`.